### ISSUE TITLE
Making fruit bowl causes an explosion

### ISSUE THREAD

**Issue by DoomAngelBlade** [NovaSector/NovaSector#3080]
Reporting client version: 515.1640
## Issue Summary

Title.

## Round ID:
11052

## Testmerges:
- [[PORT] [SEMI-MODULAR] [JANK] Ports the lovely Character Directory which was ported from Coyote Bayou](https://github.com/NovaSector/NovaSector/pull/183)
- [Ooey Gooey Slimegirl Buffs Vol. 1](https://github.com/NovaSector/NovaSector/pull/1661)
- [[SEMI-MODULAR] Taur rework Pt. 2: Quadruped taurs (Part one)](https://github.com/NovaSector/NovaSector/pull/1976)
- [Flashbangs now inform user if the security level is green, and during green alerts they must be ctrl+clicked to arm](https://github.com/NovaSector/NovaSector/pull/2553)
- [[TESTING ONLY] Debugging overlay cap being reached with taur parts](https://github.com/NovaSector/NovaSector/pull/2754)
- [[SEMI-MODULAR] Telecomms Specialist](https://github.com/NovaSector/NovaSector/pull/2757)
- [Pipe Dreams: Disposals is actually usable as an escape again](https://github.com/NovaSector/NovaSector/pull/2852)
- [The End of the Galactic Materials Market: Economy shakeup](https://github.com/NovaSector/NovaSector/pull/2498)
- [Doubles SMES output and input ability per tier, QOL Buff to Solar power generation](https://github.com/NovaSector/NovaSector/pull/3005)

## Reproduction:

Get default ingredients for fruit bowl salad.
Make it.
Lose limbs and be near dead.

**Comment by iansdoor** [comment]
indeed this is from a melon bowl
![Screenshot_199](https://github.com/NovaSector/NovaSector/assets/109386467/a3e1a3b5-5341-4be4-8a24-2fce1d6dd35e)

**Comment by lessthnthree** [comment]
There.

https://github.com/NovaSector/NovaSector/pull/3148

**Comment by Floofies** [comment]
Fixed by #3148

**Comment by Floofies** [comment]
Wow I legitimately thought it got fixed. Oops!

**Comment by Floofies** [comment]
Reportedly part of upstream issue tgstation/tgstation#81226

**Comment by projectkepler-ru** [comment]
Yes, but no fix is likely to be implemented unless someone has the willpower to do the desired change
